### PR TITLE
workgroupStorageUsed is sum of values, each rounded up to multiple of 16

### PR DESCRIPTION
**Pull request by dneto0**
Fixes: #3485

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -7114,7 +7115,7 @@ dictionary GPUComputePipelineDescriptor : GPUPipelineDescriptorBase {
                         - |layout| must be [$valid to use with$] |this|.
                         - [$validating GPUProgrammableStage$]({{GPUShaderStage/COMPUTE}},
                             |descriptor|.{{GPUComputePipelineDescriptor/compute}}, |layout|) must succeed.
-                        - Let |workgroupStorageUsed| be the sum of the [$SizeOf$](|T|) over each
+                        - Let |workgroupStorageUsed| be the sum of [=roundUp=](16,[$SizeOf$](|T|)) over each
```
Comment (kainino0x) [inline spec/index.bs:7118]:
```suggestion
                        - Let |workgroupStorageUsed| be the sum of [=roundUp=](16, [$SizeOf$](|T|)) over each
```

**Review by kainino0x — APPROVED** [review]
LGTM, thanks! Landing now as resolved by the WGSL group.

**Comment by kdashg** [comment]
<details><summary><a href="https://github.com/gpuweb/gpuweb/wiki/gpu-web-meeting-2022-10-12-13-apac-timed#tacit-resolution-queue">GPU Web Meeting 2022-10-12/13 APAC-timed</a></summary>

* workgroupStorageUsed is sum of values, each rounded up to multiple of 16 [#3522](https://github.com/gpuweb/gpuweb/pull/3522)
    * Handles the fact that padding between variables in workgroup storage class is implementation-defined. FYI only - already approved by WGSL group.
    * Backends don't define how they pack variables into workgroup memory, so this is a conservative estimate of how much space we'll use.
* MM: if workgroup variable is a struct, will padding be involved?
* KN: reason for picking 16 - it's the largest natural alignment you'd need.
* MM: so struct containing 4 floats - 16 bytes, not 128?
* KN: yes.
* KN: this was discussed by folks who understand Vulkan's constraints.

</details>

**Comment by lokokung** [comment]
Removing 'needs-cts-issue' with filing of CTS re-review draft issue named: `val: compute_pipeline:limits,workgroup_storage_size`

**Pull request merged**